### ISSUE TITLE
Keyboard Freezes

### ISSUE THREAD

**Issue by dominait** [microsoft/vscode#105609]
Issue Type: <b>Bug</b>

I have all of my extensions disabled, running a clean install of VSCode no changes to the config file and whenever I make an error that prompts a squiggly line under code js, css, package.json, files etc the keyboard freezes up and I can go back with the back arrow but thats it. I have to restart then I am good till the next error.

I can duplicate this.

Regards
Jim

VS Code version: Code 1.48.2 (a0479759d6e9ea56afa657e454193f72aef85bd0, 2020-08-25T10:09:08.021Z)
OS version: Darwin x64 19.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-9600K CPU @ 3.70GHz (6 x 3700)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 1, 1|
|Memory (System)|16.00GB (8.17GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>Extensions: none

**Comment by alexdima** [comment]
@dominait Could you please explain in greater detail the exact steps that you are doing? I have every day squiggles in the editor and I can continue typing, so I cannot reproduce.

**Comment by dominait** [comment]
I disabled all the extensions and was running the latest version of VSCode on MAC OS Catalina. If I made a code error the keyboard would freeze. I could use the back arrow and that was it. I eventually reinstalled my system again as I was also getting permission errors in the terminal that were not related to VSCode. 
Once the clean install was complete VSCode and all the extension I had were working fine. Ghost in the machine?
Regards
